projects/binutils/project.yaml

```diff
@@ -0,0 +1,3 @@
+homepage: "https://www.gnu.org/software/binutils/"
+primary_contact: "bug-binutils@gnu.org"
+auto_ccs : "p.antoine@catenacyber.fr"
```

Comment:
not sure if it works this way, better do:

```
auto_ccs:
 - <email>
```

Reply:
It has been working so far, for instance : https://github.com/google/oss-fuzz/blob/master/projects/capstone/project.yaml#L3

So, should I change it ?

Reply:
Fine to leave, since it works.